py3status/modules/hamster.py

```diff
@@ -27,19 +25,15 @@ class Py3status:
     format = '{current}'
 
     def hamster(self):
-        cur_task = check_output(shlex.split('hamster current'))
-        cur_task = cur_task.decode('ascii', 'ignore').strip()
+        cur_task = self.py3.command_output('hamster current')
         if cur_task != 'No activity':
             cur_task = cur_task.split()
             time_elapsed = cur_task[-1]
             cur_task = cur_task[2:-1]
             cur_task = "%s (%s)" % (" ".join(cur_task), time_elapsed)
```

Comment:
this will now break as we may have unicode output.  Before any unicode was being stripped out so things didn't break.
The line should be ```cur_task = u"%s (%s)" % (" ".join(cur_task), time_elapsed)``` note the `u`

I'm not testing this.  Can you do a test with a unicode task